dashboard/config/levels/custom/panels/mix-move-ai-panels-code-generation.level

```diff
@@ -11,49 +11,56 @@
     "panels": [
       {
         "text": "### Behind the scenes\n\nHow does AI generate code from your words?",
-        "imageUrl": "https://images.code.org/8b6420bcb71cdfde2997b6575a7034c8-behind the scene.gif",
+        "imageUrl": "https://images.code.org/e9fe982a414e14e551e8b7cd27561cfd-Code Gen Panel1.jpg",
         "key": "Generate_dancer_info-e349bed8-89d6-4efc-8a0e-29b8a1e5afb1",
-        "layout": "text-bottom-center"
+        "layout": "text-bottom-center",
+        "fadeInOverPrevious": true
       },
       {
         "text": "**AI went through a training process**\n\nIn training, it read huge amounts of text, like books, computer programs, and articles from the internet.",
-        "imageUrl": "https://images.code.org/85951b59bbc4cc06c971712bf91da1d9-llms panel 2.gif",
+        "imageUrl": "https://images.code.org/1363cdb9262a72d9fafc98e776fc9e54-Code Gen Panel2.jpg",
         "key": "Generate_dancer_more_details-0bf87b05-fdcb-4acc-b8d2-5455e69d6e87",
-        "layout": "text-bottom-center"
+        "layout": "text-bottom-center",
+        "fadeInOverPrevious": true
       },
       {
         "text": "**AI learned from examples**\n\nAI noticed patterns in how words usually go together and learned to predict the next word.",
-        "imageUrl": "https://images.code.org/f07aac64ce1d0fcf2e1a3798b39a72bb-llm panel3.gif",
+        "imageUrl": "https://images.code.org/2f7eb200a4b6788f8283c47190ee6a7e-Code Gen Panel3.jpg",
         "key": "Generate_dancer_more_details-48f5d03d-6aff-4230-bbc0-75fbac278d9c",
-        "layout": "text-bottom-center"
+        "layout": "text-bottom-center",
+        "fadeInOverPrevious": true
       },
       {
         "text": "**AI makes predictions**\n\nWhen you give AI some starting words, it predicts how to respond, one word after another.",
-        "imageUrl": "https://images.code.org/2df2eadd2ab8d72a58245b7c4454645f-llm panel 4.jpg",
+        "imageUrl": "https://images.code.org/f9ec60227b7ab8a245dc235b2a3e85b1-Code Gen Panel4.jpg",
         "key": "Generate_dancer_more_details-f7defbff-6781-4fcd-ba98-bb4e03422fd5",
-        "layout": "text-bottom-center"
+        "layout": "text-bottom-center",
+        "fadeInOverPrevious": true
       },
       {
         "text": "**AI generates something new**\n\nIn this way, it generates full sentences, whole conversations, and even Music Lab code!",
-        "imageUrl": "https://images.code.org/5f07dcd1ddde8a1aefe85b2dc448151d-llm panel 5b.jpg",
+        "imageUrl": "https://images.code.org/edbc9f6ff18eda312a2d882f95ec06fc-Code Gen Panel5.jpg",
         "key": "Generate_dancer_more_details-1af42de1-d2bc-4653-9e86-16d56ff2ceb3",
-        "layout": "text-bottom-center"
+        "layout": "text-bottom-center",
+        "fadeInOverPrevious": true
       },
       {
         "text": "**AI isn’t magic…it’s math**\n\nRemember, AI doesn’t think or feel — it just uses math to make predictions based on its training data.",
```

Comment:
Do you want to add a space after these ellipses?

Reply:
Good catch.  Updated.